src/test/resources/regressions/issues/000006.gobra

```diff
@@ -0,0 +1,3609 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
```

Comment:
I must confess that I was kinda scared by the size of this PR when I first saw it... until I saw this file xD

Does this example used to be problematic in the old parser? Otherwise, we can keep it for shits and giggles